### ISSUE TITLE
Make pdfmm available as vcpkg port

### ISSUE THREAD

**Issue by derreisende77** [pdfmm/pdfmm#30]
This would advertise pdfmm as an alternative for podofo and help attracting contributors maybe :)

**Comment by ceztko** [comment]
Interesting, I didn't know about vcpkg! It's not a priority, though, because there are ongoing talks (also non public) about merging[1] pdfmm back to PoDoFo again.

[1] https://www.mail-archive.com/podofo-users@lists.sourceforge.net/msg04719.html

**Comment by derreisende77** [comment]
As a user of podofo this would be great but I doubt that this will happen soon. I have followed their mailing lists for some time and they seem to be really reluctant to use c++11 or better as it seems to break some embedded systems. And their release cycle is way too slow. For example there have been some patches that fix "critical" bugs in the library after 0.9.7 but haven´t released a 0.9.8 release. I hope I will not have to wait another two years for it 😄 

In the meantime it still would be great to have vcpkg support as well for those who want to use your version. If both will get merged in the future there should not be a too big problem.

**Comment by ceztko** [comment]
If you follow the PoDoFo mailing-list something is moving, despite this is taking a lot of efforts on my side (but it was to be expected). It seems that an unofficial PoDoFo port for vcpkg is already existing[1]. We could support vcpkg officially once pdfmm is merged back to PoDoFo, which I hope it will happen later on this year.

[1] https://github.com/microsoft/vcpkg/tree/master/ports/podofo
